src/amo/components/SearchForm/Suggestion.js

```diff
@@ -4,6 +4,8 @@ import React from 'react';
 import Icon from 'ui/components/Icon';
 import LoadingText from 'ui/components/LoadingText';
 
+import './Suggestion.scss';
```

Comment:
alternatively, you could move the whole thing into a subdirectory: `components/SearchForm/Suggestion/index.js`

Reply:
Yeah I'm cool with that. Or even just making it a top-level component. I don't think there's anything from with `SearchSuggestion` and I don't like nesting components; a big list is still easier to search than a nested one.